theme/templates/base.html

```diff
@@ -196,7 +196,7 @@
                         <h3>{{ siteconf.col1_heading }}</h3>
                     </div>
                     <div class="content">
-                        {{ siteconf.col1_content|richtext_filters|safe }}
+                        {{ siteconf.col1_content|richtext_filters }}
```

Comment:
![site-conf-escaped](https://cloud.githubusercontent.com/assets/46069/25661515/f848a9c8-2fde-11e7-9a8c-4f186c43b03d.png)

These are causing formatting issues. I did not check all of the other ones. I don't know much about siteconf content, but if that is only accessible by admins, it's likely not important to remove the `|safe filters`.